types/sharp/index.d.ts

```diff
@@ -494,25 +494,25 @@ declare namespace sharp {
         toFile(fileOut: string): Promise<OutputInfo>;
 
         /**
-         * Write output to a Buffer. JPEG, PNG, WebP, AVIF, TIFF and RAW output are supported.
-         * By default, the format will match the input image, except GIF and SVG input which become PNG output.
+         * Write output to a Buffer. JPEG, PNG, WebP, AVIF, TIFF, GIF and RAW output are supported.
+         * By default, the format will match the input image, except SVG input which becomes PNG output.
```

Comment:
Hey all, this is my first contribution. Apologies if it is not up to standard! I just wanted to update the doc comments so that they match the docs of the package (link to the docs is provided in the PR template).